extensions/wikia/AdEngine/js/lookup/rubicon/rubiconTier.js

```diff
@@ -38,8 +39,20 @@ define('ext.wikia.adEngine.lookup.rubicon.rubiconTier', [
 		return sizeId + '_tier' + math.leftPad(tier, 4);
 	}
 
-	function parsePrice(tier) {
-		var matches = /^\d+_tier(\d+)/g.exec(tier);
+	function parseOpenMarketPrice(tier) {
+		if (tier.indexOf(privateTierPostfix) === -1) {
+			var matches = /^\d+_tier(\d+)/g.exec(tier);
+
+			if (matches && matches[1]) {
+				return parseInt(matches[1], 10);
```

Comment:
something I learnt hard way today - if you pass to Math.max something that is not numeric value as one of the arguments the result would be NaN so make sure to always return a number from those functions.